appveyor.yml

```diff
@@ -11,21 +11,26 @@ branches:
   only:
     - master
 
-# caching vendor/bundle appears to break horribly in the face of gems checked out of git
-# cache:
-#   - vendor/bundle
+environment:
+  matrix:
+    - ruby_version: "25"
+
+clone_folder: c:\projects\berkshelf
+clone_depth: 1
 
 install:
-  - ps: iex (irm https://omnitruck.chef.io/install.ps1); Install-Project -Project chefdk -channel stable
-  - bundle config --local path vendor/bundle
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - SET CI=true
   - SET BUNDLE_WITHOUT=changelog:style
+  - echo %PATH%
+  - ruby --version
+  - gem --version
+  - bundler --version
```

Comment:
should probably bump the versions here unless that's handled somewhere else already.

Reply:
Done now